parrots.yaml

```diff
@@ -427,3 +427,5 @@
 - hd: hd/daftpunkparrot.gif
   name: Daft Punk Parrot
   tip: Harder Better Faster Stronger
+  - hd: hd/socialdistanceparrot.gif
```

Comment:
this and the following line should be aligned in indentation with the previous one, you have a extra tab 😓